src/less/button/ds6/button-base.less

```diff
@@ -308,4 +308,12 @@ span.expand-btn__icon {
     background-size: contain;
     flex-shrink: 0;
     margin-left: 8px;
+
+    button.expand-btn span.expand-btn__cell > &:only-child {
```

Comment:
Selector can be reduced from `button.expand-btn span.expand-btn__cell` to just `span.expand-btn__cell`... unless you need the specificty.

Reply:
Yes, they should be reduced. That is one of the advantages of using BEM syntax. `span.expand-btn__cell` will end up being a unique selector.